docs/notes/notes.rst

```diff
@@ -5,25 +5,20 @@
 Information about the release lifecycle can be found
 `here <https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md>`_.
 
-Version 2.11.1
+Version 2.11.2
 ==============
 
 This release includes the following **improvements**:
 
-    1. Correct CONTRIBUTING.md typo
-    2. Improve validation on PID_PROPERTY_LIST deserialization
-    3. Apply eProsima brand style to Fast DDS repository
-    4. Fix spelling mistake: SUBSTRACTION to SUBTRACTION
+    1. Improve Shared Memory resilience to crashing participants
+    2. User configuration for Shared Memory metatraffic
```

Comment:
Please add a cross-reference to [here](https://fast-dds.docs.eprosima.com/en/latest/fastdds/property_policies/non_consolidated_qos.html#shm-transport-meta-traffic-enforcement). I believe the tag is `property_policies_shm_enforce_metatraffic`